CRM/Core/Payment.php

```diff
@@ -518,9 +519,23 @@ public function getText($context, $params) {
             $gotText .= ' ' . ts('You will receive an email receipt for each recurring contribution.');
           }
         }
-        break;
+        return $gotText;
+
+      case 'contributionPageContinueText':
+        if ($params['amount'] <= 0) {
```

Comment:
Elsewhere we seem to compare 0.0

Reply:
Yeah - I used <0 as I was moving from this line

https://github.com/civicrm/civicrm-core/pull/13787/files#diff-fc454d9ef40ac8417cb3c056269d4cccL39